src/node_task_runner.cc

```diff
@@ -0,0 +1,275 @@
+#include "node_task_runner.h"
+#include "util.h"
+
+#include <regex>  // NOLINT(build/c++11)
+
+namespace node::task_runner {
+
+#ifdef _WIN32
+static constexpr char bin_path[] = "\\node_modules\\.bin";
```

Comment:
Ok. Why isn't this...

```suggestion
static constexpr const char* bin_path = "\\node_modules\\.bin";
```